Description: Fix requirements.txt
 This patch avoids FTBFS with dpkg-gencontrol being confused by the !=
 stuff.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-07-15

--- heat-7.0.0~b2.orig/requirements.txt
+++ heat-7.0.0~b2/requirements.txt
@@ -5,14 +5,14 @@
 pbr>=1.6 # Apache-2.0
 Babel>=2.3.4 # BSD
 croniter>=0.3.4 # MIT License
-cryptography!=1.3.0,>=1.0 # BSD/Apache-2.0
+cryptography
 debtcollector>=1.2.0 # Apache-2.0
-eventlet!=0.18.3,>=0.18.2 # MIT
+eventlet
 greenlet>=0.3.2 # MIT
 keystoneauth1>=2.10.0 # Apache-2.0
-keystonemiddleware!=4.1.0,!=4.5.0,>=4.0.0 # Apache-2.0
+keystonemiddleware
 lxml>=2.3 # BSD
-netaddr!=0.7.16,>=0.7.13 # BSD
+netaddr
 oslo.cache>=1.5.0 # Apache-2.0
 oslo.config>=3.14.0 # Apache-2.0
 oslo.concurrency>=3.8.0 # Apache-2.0
@@ -34,17 +34,17 @@ pycrypto>=2.6 # Public Domain
 aodhclient>=0.5.0 # Apache-2.0
 python-barbicanclient>=4.0.0 # Apache-2.0
 python-ceilometerclient>=2.5.0 # Apache-2.0
-python-cinderclient!=1.7.0,!=1.7.1,>=1.6.0 # Apache-2.0
+python-cinderclient
 python-designateclient>=1.5.0 # Apache-2.0
 python-glanceclient!=2.4.0,>=2.3.0 # Apache-2.0
 python-heatclient>=1.4.0 # Apache-2.0
-python-keystoneclient!=2.1.0,>=2.0.0 # Apache-2.0
+python-keystoneclient
 python-magnumclient>=2.0.0 # Apache-2.0
 python-manilaclient>=1.10.0 # Apache-2.0
 python-mistralclient>=2.0.0 # Apache-2.0
 python-monascaclient>=1.1.0 # Apache-2.0
 python-neutronclient>=5.1.0 # Apache-2.0
-python-novaclient!=2.33.0,>=2.29.0 # Apache-2.0
+python-novaclient
 python-openstackclient>=2.1.0 # Apache-2.0
 python-saharaclient>=0.18.0 # Apache-2.0
 python-senlinclient>=0.3.0 # Apache-2.0
@@ -54,9 +54,8 @@ python-zaqarclient>=1.0.0 # Apache-2.0
 pytz>=2013.6 # MIT
 PyYAML>=3.1.0 # MIT
 requests>=2.10.0 # Apache-2.0
-retrying!=1.3.0,>=1.2.3 # Apache-2.0
-Routes!=2.0,!=2.1,!=2.3.0,>=1.12.3;python_version=='2.7' # MIT
-Routes!=2.0,!=2.3.0,>=1.12.3;python_version!='2.7' # MIT
+retrying
+Routes
 six>=1.9.0 # MIT
 SQLAlchemy<1.1.0,>=1.0.10 # MIT
 sqlalchemy-migrate>=0.9.6 # Apache-2.0
